extensions/wikia/Listusers/SpecialListusers_hooks.php

```diff
@@ -24,23 +24,32 @@ public static function Activeusers( &$list ) {
 		return true;
 	}
 
-	/**
-	 * update list users table on user right change
-	 *
-	 * @see https://www.mediawiki.org/wiki/Manual:Hooks/UserRights
-	 *
-	 * @author      Piotr Molski <moli@wikia-inc.com>
-	 * @version     1.0.0
-	 * @param       User    $user object
-	 * @param       array   $addgroup - selected groups for user
-	 * @param       array   $removegroup - disabled groups for user
-	 * @return bool
-	 */
-	static public function updateUserRights( User $user, array $addgroup, array $removegroup ) {
-		global $wgCityId;
 
-		$data = new ListusersData($wgCityId);
-		$data->updateUserGroups( $user, $addgroup, $removegroup );
-		return true;
+	static public function updateUserRights( User $user ) {
+		if ( !$user->isAllowed( 'bot' ) ) {
+			$listUsersUpdate = new ListUsersUpdate();
+			$listUsersUpdate->setUserId( $user->getId() );
+			$listUsersUpdate->setUserGroups( $user->getGroups() );
+
+			$task = UpdateListUsersTask::newLocalTask();
+			$task->call( 'updateUserGroups', $listUsersUpdate );
+			$task->queue();
+		}
+	}
+
+	public static function doEditUpdate( WikiPage $wikiPage, Revision $revision, $baseRevId, User $user ) {
+		if ( $user->isAnon() || $user->isAllowed( 'bot' ) ) {
+			return;
+		}
```

Comment:
Are you sure that we want to skip bot accounts? http://poznan.wikia.com/wiki/Specjalna:U%C5%BCytkownicy lists bots...

Reply:
seems like legacy script had this option for no reason - I'll remove it